README.md

```diff
@@ -226,166 +121,9 @@ We use the [Development Project Board](https://github.com/orgs/getAlby/projects/
 - Have a look at this Readme. Can it be improved? Do you see typos? You can open a PR or reach out to us in [our community chat](https://bitcoindesign.slack.com/archives/C02591ADXM2).
 - You can help with [translations](#translations)
 
-### Creating a PR
-
-When creating a PR please take this points as a reminder:
-
-- If there's not yet an issue for your PR please first [create an issue](https://github.com/bumi/lightning-browser-extension/issues/new) with a proposal what you would like to do. This allows us to give feedback and helps you no wasting time and motivation
-- Think in iterations (babysteps)\
-  You can always start a PR and if you feel like adding on more things to it, better branch off and [create a new i.e. _draft_-PR](https://github.blog/2019-02-14-introducing-draft-pull-requests/)
-- Newly added components should have a unit-test
-- If you work on a more complex PR please [join our community chat](https://bitcoindesign.slack.com/archives/C02591ADXM2) to get feedback, discuss the best way to tackle the challenge, and to ensure that there's no duplication of work. It's often faster and nicer to chat or call about questions than to do ping-pong comments in PRs
-
-### Code format & preferences
-
-- [ESLint](https://eslint.org/) for code linting
-- [Prettier](https://prettier.io/) for code (and more) formatting
-- We prefer [Axios](https://axios-http.com/) over [fetch](https://developer.mozilla.org/en-US/docs/Web/API/Fetch_API)
+[Contribution guide for new developers.](./doc/Contribution.md)
```

Comment:
Should this be moved to the developers section above?

Reply:
Let me change it to see the effect